src/main/scala/slamdata/SbtSlamdata.scala

```diff
@@ -73,7 +82,7 @@ object SbtSlamData extends AutoPlugin {
         "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases",
         "bintray/non" at "http://dl.bintray.com/non/maven"),
       addCompilerPlugin("org.spire-math"  %% "kind-projector" % "0.9.3"),
-      addCompilerPlugin("org.scalamacros" %  "paradise"       % "2.1.0" cross CrossVersion.full),
+      addCompilerPlugin("org.scalamacros" %  "paradise"       % "2.1.0" cross CrossVersion.fullMapped(stripTLSVersion)),
```

Comment:
`patch`, [you say](http://www.scala-sbt.org/0.13/sxr/sbt/CrossVersion.scala.html)?